src/sentry/runner/commands/presenters/slackpresenter.py

```diff
@@ -0,0 +1,125 @@
+from typing import List, Tuple
+
+import requests
+
+from sentry.conf.server import OPTIONS_AUTOMATOR_SLACK_WEBHOOK_URL
+from sentry.runner.commands.presenters.optionspresenter import OptionsPresenter
+from sentry.utils import json
+
+
+class SlackPresenter(OptionsPresenter):
+    """
+    Sends changes of runtime options made via sentry configoptions
+    to a webhook url in a truncated json format. The webhook url can
+    be configured to your liking, but for this use case it is ideally
+    integrated with Slack.
+    """
+
+    MAX_OPTION_VALUE_LENGTH = 30
+
+    def __init__(self) -> None:
+        self.drifted_options: List[Tuple[str, str]] = []
+        self.channel_updated_options: List[str] = []
+        self.updated_options: List[Tuple[str, str, str]] = []
+        self.set_options: List[Tuple[str, str]] = []
+        self.unset_options: List[str] = []
+        self.error_options: List[Tuple[str, str]] = []
+        self.unregistered_options: List[str] = []
+        self.invalid_type_options: List[Tuple[str, str, str]] = []
+
+    @staticmethod
+    def is_slack_enabled():
+        if OPTIONS_AUTOMATOR_SLACK_WEBHOOK_URL == "":
+            return False
+
+        try:
+            test_payload: dict = {
+                "drifted_options": [],
+                "updated_options": [],
+                "set_options": [],
+                "unset_options": [],
+                "error_options": [],
+                "unregistered_options": [],
+                "invalid_type_options": [],
+            }
+
+            response = requests.post(OPTIONS_AUTOMATOR_SLACK_WEBHOOK_URL, json=test_payload)
+            if response.status_code == 200:
+                return True
```

Comment:
Isn't this going to actually send a message ?

Reply:
no, because its empty, and the way the webhook is configured will not post if its empty updates

Reply:
Then here, if the automator is configured for talking to Slack but the webhook does not work you should not just transparently fail. 
I'd say retry twice and if it is still not working raise exception and make the automator crash.

If you did not make it crash loudly, we would just see notifications suddenly disappear with no hint on the issue and no way to troubleshoot.